### DISCUSSION TITLE
How do I use dbt YAML Selectors?

### DISCUSSION THREAD

**Discussion by gnilrets** [dagster-io/dagster#20417] [Q&A] [answered]
dbt has [YAML selectors](https://docs.getdbt.com/reference/node-selection/yaml-selectors).  How do we define our asset selection in dagster to include just those assets defined by the selector?

**Comment by rexledesma** [comment, marked as answer]
It's not supported yet. Feel free to file a feature request.

**Reply by garethbrickman** [reply]
https://github.com/dagster-io/dagster/issues/22252

**Reply by garethbrickman** [reply]
Resolved with: https://github.com/dagster-io/dagster/pull/29554!

You can now use:

```python
@dbt_assets(..., selector="some_selector")
def my_dbt_assets(): ...
```

to specify the set of assets to include based off of a selector, or pass a selector argument into build_dbt_manifest_asset_selection.

Support for this is in version `1.10.13`
